test/validators.js

```diff
@@ -986,100 +1012,118 @@ describe('Validators', function () {
+                , '+27821231234'
+                , '27821231234'
+            ]
+            , invalid: [
+                '082123'
+                , '08212312345'
+                , '21821231234'
+                , '+21821231234'
+                , '+0821231234'
+            ],
+            args: ['en-ZA']
+        });
 
-      test({
-        validator: 'isMobilePhone'
-        , valid: [
-            '61404111222'
-          , '+61411222333'
-          , '0417123456'
-        ]
-        , invalid: [
-            '082123'
-          , '08212312345'
-          , '21821231234'
-          , '+21821231234'
-          , '+0821231234'
-        ],
-        args: ['en-AU']
-      });
+        test({
+            validator: 'isMobilePhone'
+            , valid: [
+                '61404111222'
+                , '+61411222333'
+                , '0417123456'
+            ]
+            , invalid: [
+                '082123'
+                , '08212312345'
+                , '21821231234'
+                , '+21821231234'
+                , '+0821231234'
+            ],
+            args: ['en-AU']
+        });
 
-      test({
-        validator: 'isMobilePhone'
-        , valid: [
-            '0612457898'
-          , '+33612457898'
-          , '33612457898'
-          , '0712457898'
-          , '+33712457898'
-          , '33712457898'
-        ]
-        , invalid: [
-            '061245789'
-          , '06124578980'
-          , '0112457898'
-          , '0212457898'
-          , '0312457898'
-          , '0412457898'
-          , '0512457898'
-          , '0812457898'
-          , '0912457898'
-          , '+34612457898'
-          , '+336124578980'
-          , '+3361245789'
-        ]
-        , args: ['fr-FR']
-      });
+        test({
+            validator: 'isMobilePhone'
+            , valid: [
+                '0612457898'
+                , '+33612457898'
+                , '33612457898'
+                , '0712457898'
+                , '+33712457898'
+                , '33712457898'
+            ]
+            , invalid: [
+                '061245789'
+                , '06124578980'
+                , '0112457898'
+                , '0212457898'
+                , '0312457898'
+                , '0412457898'
+                , '0512457898'
+                , '0812457898'
+                , '0912457898'
+                , '+34612457898'
+                , '+336124578980'
+                , '+3361245789'
+            ]
+            , args: ['fr-FR']
+        });
+
+        test({
+            validator: 'isMobilePhone'
+            , valid: [
+                '2102323234'
+                , '+302646041461'
+                , '+306944848966'
+                , '6944848966'
+            ]
+            , invalid: [
+                '120000000'
+                , '20000000000'
+                , '68129485729'
+                , '6589394827'
+                , '298RI89572'
+            ]
+            , args: ['el-GR']
+        });
```

Comment:
I show it too this afternoon and pushed a new commit ... :)

2015-02-21 2:33 GMT+02:00 chriso <notifications@github.com>:

> In test/validators.js
> <https://github.com/chriso/validator.js/pull/364#discussion_r25115343>:
>
> > +            validator: 'isMobilePhone'
> > +            , valid: [
> > +                '2102323234'
> > +                , '+302646041461'
> > +                , '+306944848966'
> > +                , '6944848966'
> > +            ]
> > +            , invalid: [
> > +                '120000000'
> > +                , '20000000000'
> > +                , '68129485729'
> > +                , '6589394827'
> > +                , '298RI89572'
> > +            ]
> > +            , args: ['el-GR']
> > +        });
>
> Actually I see them here, never mind.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/chriso/validator.js/pull/364/files#r25115343>.
>



-- 
Γιώργος